qualification/recv.py

```diff
@@ -199,20 +200,60 @@ async def next_complete_chunk(
         ----------
         min_timestamp, max_delay
             See :meth:`complete_chunks`
+        timeout
+            Maximum time to wait
+
+        Raises
+        ------
+        TimeoutError
+            if a complete chunk is not received in time
+        RuntimeError
+            if the stream is stopped before a complete chunk is received
```

Comment:
I will admit, I would've liked the first letter of these sentences to be capitalised (like other uses/implementations). Don't feel strongly enough to request a single commit just for that, but if you do enact my other docstring suggestion please do tend to this as well.